cmd/tools/rest/rest.go

```diff
@@ -585,10 +593,9 @@ func init() {
 
 	showFlags := showCmd.Flags()
 	showFlags.StringVarP(&args.API, "api", "a", "", "REST API PATTERN to show")
-	showFlags.StringVar(&args.Endpoint, "endpoint", "", "By default, /api is appended to passed argument in --api. Use --endpoint instead to pass absolute path of url")
 	showFlags.BoolVar(&args.DownloadAll, "all", false, "Collect all records by walking pagination links")
 	showFlags.BoolVarP(&args.Verbose, "verbose", "v", false, "Be verbose")
-	showFlags.StringVarP(&args.MaxRecords, "max-records", "m", "", "Limit the number of records returned before providing pagination link")
+	showFlags.IntVarP(&args.MaxRecords, "max-records", "m", -1, "Limit the number of records returned before providing pagination link")
```

Comment:
-1 is not a legal value.

`"message": "The value \"-1\" is invalid for field \"max_records\" (<1..1000000>)",`